### PR TITLE
Implementation of minctoolpaths.txt configuration file.

### PR DESCRIPTION
**Pull request by nist-anka**

### CODE REVIEW

File: IbisLib/filereader.cpp [hunk 1]
```diff
@@ -85,6 +86,36 @@ QString FindExecutable( QStringList & candidatePaths )
 
 bool FileReader::FindMincConverter()
 {
+    if( !m_mincconvert.isEmpty() && !m_minccalc.isEmpty() )
+        return true;
+    // First check if there is a configuration file
+    QString mincconfig = QDir::homePath() + "/" + IBIS_CONFIGURATION_SUBDIRECTORY + "/minctoolspath.txt";
```
Comment (drouin-simon) [inline IbisLib/filereader.cpp]: Call Application::GetConfigDirectory() instead

Comment (drouin-simon) [reply IbisLib/filereader.cpp]: I think that if the file is present, it should overwrite the list of search paths and not be appended.

**Comment by nist-anka** [comment]
It is not appended, if  it contains a valid path, the other paths are not
searched. Anyway, I'd like to discuss with you some  related issue.

File: IbisLib/filereader.cpp (hunk 1 above)
Comment (nist-anka) [reply IbisLib/filereader.cpp]: fileReader does not know application

**Pull request closed without merging**